testing/environments/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Basic debian file with curl, wget and nano installed to fetch files
 # an update config files
 FROM debian:latest
-MAINTAINER Nicolas Ruflin <ruflin@elastic.co>
+LABEL org.opencontainers.image.authors="Nicolas Ruflin <ruflin@elastic.co>"
```

Comment:
Nothing against updating the Docker file without changing the information, but Ruflin is not the maintainer of this dockerfle any more. I'd delete this entry because it's a dockerfile just for testing.